scripts/changelog/generate.mjs

```diff
@@ -0,0 +1,70 @@
+import fs from 'node:fs';
+import addStream from 'add-stream';
+import chalk from 'chalk';
+import conventionalChangelog from 'conventional-changelog';
```

Comment:
FYI the package I mentioned earlier: https://github.com/antfu/changelogithub. It creates changelogs like https://github.com/unocss/unocss/releases/tag/v0.39.0.